<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Massimiliano Torromeo</author>
 <class>DlgFuzzyOpen</class>
 <widget class="QWidget" name="DlgFuzzyOpen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fuzzy Open</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QEventLineEdit" name="txtFilter"/>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="btnSettings">
       <property name="toolTip">
        <string>Settings</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListWidget" name="listUrl"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblProgress">
     <property name="text">
      <string/>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
     <property name="indent">
      <number>2</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEventLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qeventlineedit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
